nilearn/datasets/utils.py

```diff
@@ -531,7 +537,10 @@ def _fetch_file(url, data_dir, resume=True, overwrite=False,
         dt = time.time() - t0
         if verbose > 0:
             # Complete the reporting hook
-            sys.stderr.write(' ...done. (%i seconds, %i min)\n' % (dt, dt // 60))
+            sys.stderr.write(' ...done. ({0:.4f} MB, {1:.0f} seconds, '
```

Comment:
> Maybe 2 is good enough.

That's I used initially, but with very small data, you can have something like 0.0MB downloaded. Maybe a more clever string (using KB, MB, GB) depending on the size could be better.